src/services/textChanges.ts

```diff
@@ -384,8 +384,8 @@ namespace ts.textChanges {
             }
         }
 
-        public insertNodeBefore(sourceFile: SourceFile, before: Node, newNode: Node, blankLineBetween = false): void {
-            this.insertNodeAt(sourceFile, getAdjustedStartPosition(sourceFile, before, {}), newNode, this.getOptionsForInsertNodeBefore(before, newNode, blankLineBetween));
+        public insertNodeBefore(sourceFile: SourceFile, before: Node, newNode: Node, blankLineBetween = false, options = {}): void {

```

Comment:
I think `options` needs a type annotation, since `{}` probably isn’t the type you want.